stdlib/README.md

```diff
@@ -0,0 +1,20 @@
+# nix-stdlib
```

Comment:
I would love to have something like that as an project on it's own.

Reply:
Should we create yet another repo?